bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-na-mobile.html

```diff
@@ -0,0 +1,56 @@
+{#
```

Comment:
**Question:** is there any reason why this couldn't extend `firefox/whatsnew/index.html` instead of duplicate it again?